### ISSUE TITLE
1.0.28 has a dependency on "publish" package

### ISSUE THREAD

**Issue by vapier** [PeculiarVentures/ByteStream.js#5]
this change isn't in git.  was it an accidental local modification ?  there's no need for this dep that i can see, and it's causing a lot more unnecessary stuff from getting pulled in (e.g. the `npm` package).
```
   "jsnext:main": "./src/bytestream.js",
   "module": "./src/bytestream.js",
   "main": "./build/bytestream.js",
-  "license": "BSD-3-Clause"
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "publish": "^0.6.0"
+  }
 }
```

**Comment by YuryStrozhevsky** [comment]
@vapier Please check 1.0.29 - hope it would fix both your issues

**Comment by awattez** [comment]
Thx it's fixed

**Comment by awattez** [comment]
Thx it's fixed issues #4 and #5
